SPECS/firewalld/firewalld.spec

```diff
@@ -3,7 +3,7 @@
 Summary:        A firewall daemon with D-Bus interface providing a dynamic firewall
 Name:           firewalld
 Version:        2.0.2
-Release:        3%{?dist}
+Release:        2%{?dist}
```

Comment:
When I upgraded python to 3.12, it broke a handful of packages due to missing __pycache__ files. I updated those packages in the same PR - see https://github.com/microsoft/azurelinux/pull/7858/files#diff-4b714dcc3b08c555d4b13cea1e3c8790f4253060696205fda66e3bd3d83df107. But since this PR fixes the pycache issues, I decided to revert all of the temporary fixes